docs/5-build-experiment.rst

```diff
@@ -19,17 +19,16 @@ Ramble will build the source code and set up the following workspace directory s
     experiments_root/
         ramble/
         spack/
-        <benchmark>/
-            <ProgrammingModel>/
-                <system>/
-                    workspace/
-                        configs/
-                            (everything from source/configs/<system>)
-                            (everything from source/experiments/<benchmark>)
+        <Benchmark/ProgrammingModel>/
+            <System>/
+                workspace/
+                    configs/
+                        (everything from source/configs/<System>)
+                        (everything from source/experiments/<Benchmark/ProgrammingModel>)
                         experiments/
```

Comment:
(issue) This `experiments` dir (and all subdirs of it) should be dedented one level (to be aligned with `configs/`.